pom.xml

```diff
@@ -287,26 +287,73 @@
 					<port>8989</port>
 				</configuration>
 			</plugin>
+
 			<plugin>
-				<groupId>com.github.warmuuh</groupId>
-				<artifactId>libsass-maven-plugin</artifactId>
-				<version>0.2.9-libsass_3.4.7</version>
+				<groupId>com.github.eirslett</groupId>
+				<artifactId>frontend-maven-plugin</artifactId>
```

Comment:
Pretty handy plugin to include Node/NPM/Yarn in the build process: https://github.com/eirslett/frontend-maven-plugin